src/sentry/ingest/transaction_clusterer/tasks.py

```diff
@@ -0,0 +1,36 @@
+from typing import Any
+
+import sentry_sdk
+from django.conf import settings
+
+from sentry import features
+from sentry.tasks.base import instrumented_task
+
+from . import rules
+from .datasource import redis
+from .tree import TreeClusterer
+
+#: Minimum number of children in the URL tree which triggers a merge.
+#: See TreeClusterer for more information.
+#: NOTE: We could make this configurable through django settings or even per-project in the future.
+MERGE_THRESHOLD = 100
+
+
+@instrumented_task(
+    name="sentry.ingest.transaction_clusterer.tasks.run_clusterer",
+    queue="txcluster",
+    soft_timeout=60 * 60,
+    # TODO: set appropriate soft_timeout
```

Comment:
Is this timeout for this task or the connection to Redis? Do we stop running the task when the timeout is reached?